web/src/components/application-detail-page/application-detail/index.tsx

```diff
@@ -319,7 +320,7 @@ export const ApplicationDetail: FC<ApplicationDetailProps> = memo(
             display="flex"
           >
             <ReactMarkdown linkTarget="_blank" className={classes.markdown}>
-              {app.description}
+              {description || "No description."}
```

Comment:
Why do we need this?
I feel that the current empty string is better since it can hide the unneeded description block.

Reply:
I get your point 👍  But to me, showing that there will be something there in case of the configuration file contains a description field would be better for the UI, and users who have not yet used that feature will be awarded of that

Reply:
The design is still not good enough for that empty case and we're having Play environment to show up that feature.
So I think that it should be hidden until we have a better design. 🤔

Reply:
Having a better UI is long time issue of pipecd development 😄  so well, if you guys really think that the current description is ugly, then it's okay to me to remove that if it's empty

Reply:
done, updated by https://github.com/pipe-cd/pipecd/pull/3626/commits/b36e46c04b59ac74ce9bc1f101206cceb9426ed4